### PR TITLE
Fix security error and add some UI for Query function.

### PR DESCRIPTION
**Pull request by unitech1120**
Ignore the package-lock.json file operation in dockerfiles.

### CODE REVIEW

**Comment by joerg84** [comment]
@unitech1120 Why did you close this?

File: arangopipe/arangopipe_frontend/app/src/containers/App/App.jsx [hunk 1]
```diff
@@ -55,6 +56,7 @@ class App extends React.Component{
      } else {
        this.getCurrentUser()
      }
+     console.log(this.props)
```
Comment (pluma) [inline arangopipe/arangopipe_frontend/app/src/containers/App/App.jsx]: Stray console.log here or is this intentional?

File: arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import brace from 'brace';
+import AceEditor from 'react-ace';
+
+import 'brace/theme/xcode';
+
+import AqlMode from './AqlMode'
+
+import {
+
+} from '../../actions/document'
```
Comment (pluma) [inline arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx]: This looks like it doesn't import anything. Is this intentional?

File: arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    currentQuery: ''
+  };
+
+  componentDidMount() {
+    const aqlMode = new AqlMode();
+    this.refs.aceEditor.editor.getSession().setMode(aqlMode);
+    this.refs.aceEditor.editor.getSession().setUseWrapMode(true);
+    this.refs.aceEditor.editor.resize();
+
+    console.log(this.refs.aceEditor.editor)
```
Comment (pluma) [inline arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx]: This looks like a stray debugging console.log.

File: arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  }
+}
+
+const mapStateToProps = state => ({
+  document: state.document.documents
+});
+
+const mapDispatchToProps = dispatch => bindActionCreators({
+
+}, dispatch);
```
Comment (pluma) [inline arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx]: This looks empty. Can this be omitted? mapDispatchToProps isn't a required argument as far as I know.

**Comment by pluma** [comment]
@joerg84 this seems legit, but I'd veto merging as-is because there's no way we can review all the changes to ace (although we could manually diff them against whatever official release bundle they are from). I'd recommend trying to replace ace with an alternative like brace first, so the repo no longer bundles a dependency. The other changes look valid.

**Comment by pluma** [comment]
Note also that for a proper PR someone should try the changes locally. Some of the dependency version bumps are semver major and could include breaking changes.

File: arangopipe/arangopipe_frontend/app/src/containers/App/App.jsx (hunk 1 above)
Comment (unitech1120) [reply arangopipe/arangopipe_frontend/app/src/containers/App/App.jsx]: Query functionality is in the middle of developing now. So don't pay attention to the code now. After finishing, let me clean the code.

File: arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx (hunk 2 above)
Comment (unitech1120) [reply arangopipe/arangopipe_frontend/app/src/containers/Query/Query.jsx]: There will be some action functions in the future.

**Comment by unitech1120** [comment]
Regarding the react-ace module,  I didn't change anything on module.  I just add new mode for AQL. 
At the moment, there is no builtin mode for AQL in react-ace module.  So I added AQL mode to the project not module and import it and use. React-ace module used brace initially. 
And I upgraded all dependencies to the latest version in package.json and confirmed no error and bumping on my side.

**Pull request closed without merging**